smithy-aws-iam-traits/src/main/java/software/amazon/smithy/aws/iam/traits/ConditionKeyDefinition.java

```diff
@@ -29,17 +31,25 @@ public final class ConditionKeyDefinition implements ToNode, ToSmithyBuilder<Con
     private static final String DOCUMENTATION = "documentation";
     private static final String EXTERNAL_DOCUMENTATION = "externalDocumentation";
     private static final String RELATIVE_DOCUMENTATION = "relativeDocumentation";
+    private static final String REQUIRED = "required";
 
     private final String type;
     private final String documentation;
     private final String externalDocumentation;
     private final String relativeDocumentation;
 
+    /**
```

Comment:
These should be part of the javadoc for the `isRequired` method and not on the private var.